Jenkinsfile_k8s

```diff
@@ -83,6 +83,7 @@ spec:
   stages {
     stage('Install Dependencies') {
       steps {
+        sh 'rm .nvmrc'
```

Comment:
```suggestion
        # TODO: workaround to be replaced, see https://github.com/jenkins-infra/plugin-site/pull/1182/
        sh 'asdf install nodejs 16.17.0'
```
Quick fix, tested in a replay. The current nodejs version used in infra.ci.jenkins.io for this build is defined here: https://github.com/jenkins-infra/docker-builder/blob/9ca0a78d82d28f81c2718b8cd59eb209e497f9b7/Dockerfile#L103
Better long term solutions:
- An updatecli manifest can be added to jenkins-infra/docker-builder to keep track of new NodeJS 16 versions
- There might be a way to configure asdf to look at existing legacy versions files to know which tool versions have to be installed: https://blog.logrocket.com/manage-node-js-versions-using-asdf/ (Section "Legacy support") cc @dduportal

Reply:
Renovate should be disabled for this `.nvmrc` file, in profit of one of the 2 solutions listed above.

Reply:
asdf specifically doesn't install new versions on demand. You have to tell it. I configured it to handle legacy .nvmrc for compatilbity for those not using asdf. Which is why this is failing.

If we want to have node upgrading, then we should have a step in the pipeline that does `asdf install nodejs $(cat .nvmrc)` so it downloads the right version as needed.

I don't think asdf handles semver type thing, its either an exact match or it yells but I could be wrong there.

If we update docker-builder, it'll fail for reports and jenkins.io that also uses nodejs, the good thing about asdf was being able to handle different versions between projects.

I think we just tell renovate to ignore updating node. Since there's no runtime, there's no real security risk of using an older version to build things.

Reply:
Digging a little bit more there is also [`.tool-version` config files](https://cs.github.com/?scope=org%3Ajenkins-infra&scopeName=jenkins-infra&q=tool-versions) for asdf, but I don't remember/see at the moment when/if it's loaded by asdf, will take another look after a good night.

Reply:
.tool-versions is the non legacy of .nvmrc
works for all asdf plugins, while .rvmrc and .nvmrc are just ruby and node only